code/game/objects/items/weapons/implants/implantchair.dm

```diff
@@ -79,8 +79,8 @@
 		if(!ismob(G.affecting))
 			return
 		var/mob/M = G.affecting
-		if(M.buckled_mob)
-			usr << "[M] will not fit into [src] because they have a slime latched onto their head."
+		if(M.buckled_mobs.len)
+			usr << "[M] will not fit into [src] because they have a creature on them!"

```

Comment:
Technically @phil235, if somebody has 3 creatures on them, it's still valid to say they have "a creature" on them.
I will update the ```usr```'s